dev/drivers/scripts/stats/global_ens/jevs_global_ens_cmce_atmos_precip_stats.sh

```diff
@@ -4,7 +4,7 @@
 #PBS -q dev
 #PBS -A VERF-DEV
 #PBS -l walltime=00:30:00
-#PBS -l place=vscatter,select=1:ncpus=2:mem=100GB
+#PBS -l place=vscatter,select=1:ncpus=1:mem=10GB
```

Comment:
@GwenChen-NOAA Thanks for making these resource adjustments! Can you please make the same resource adjustments in the corresponding .ecf script?

CC @malloryprow